app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -1,69 +1,82 @@
-import com.kickstarter.ui.IntentKey;
-import com.kickstarter.ui.adapters.CreatorDashboardAdapter;
+import com.kickstarter.ui.adapters.CreatorDashboardBottomSheetAdapter;
+import com.kickstarter.ui.fragments.CreatorDashboardFragment;
 import com.kickstarter.viewmodels.CreatorDashboardViewModel;
 
+import java.util.List;
+
 import butterknife.Bind;
 import butterknife.ButterKnife;
 
 import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
 
 @RequiresActivityViewModel(CreatorDashboardViewModel.ViewModel.class)
 public final class CreatorDashboardActivity extends BaseActivity<CreatorDashboardViewModel.ViewModel> {
-  private CreatorDashboardAdapter adapter;
 
-  protected @Bind(R.id.creator_dashboard_recycler_view) RecyclerView creatorDashboardRecyclerView;
+  private CreatorDashboardBottomSheetAdapter bottomSheetAdapter;
+  private BottomSheetBehavior bottomSheetBehavior;
+
+  protected @Bind(R.id.creator_dashboard_bottom_sheet_recycler_view) RecyclerView bottomSheetRecyclerView;
 
   @Override
   protected void onCreate(final @Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.creator_dashboard_layout);
     ButterKnife.bind(this);
 
-    this.adapter = new CreatorDashboardAdapter();
-    this.creatorDashboardRecyclerView.setAdapter(this.adapter);
-    this.creatorDashboardRecyclerView.setLayoutManager(new LinearLayoutManager(this));
+    // Set up the bottom sheet recycler view.
+    this.bottomSheetAdapter = new CreatorDashboardBottomSheetAdapter();
+    this.bottomSheetRecyclerView.setAdapter(this.bottomSheetAdapter);
+    this.bottomSheetRecyclerView.setLayoutManager(new LinearLayoutManager(this)); // todo: reuse LayoutManager?
+    this.bottomSheetBehavior = BottomSheetBehavior.from(this.bottomSheetRecyclerView);
 
     this.viewModel.outputs.projectAndStats()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(this::renderProjectAndStats);
+      .subscribe(this::createProjectDashboardFragments);
 
-    this.viewModel.outputs.startProjectActivity()
+    this.viewModel.outputs.projectsForBottomSheet()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(projectAndRefTag -> this.startProjectActivity(projectAndRefTag.first, projectAndRefTag.second));
+      .subscribe(this::setProjectsForDropdown);
+  }
+
+  private void setProjectsForDropdown(final @NonNull List<Project> projects) {
+    this.bottomSheetAdapter.takeProjects(projects);
   }
 
-  private void startProjectActivity(final @NonNull Project project, final @NonNull RefTag refTag) {
-    final Intent intent = new Intent(this, ProjectActivity.class)
-      .putExtra(IntentKey.PROJECT, project)
-      .putExtra(IntentKey.REF_TAG, refTag);
-    startActivity(intent);
+  private void createProjectDashboardFragments(final @NonNull Pair<Project, ProjectStatsEnvelope> projectAndStats) {
```

Comment:
sorry but I would even singularlize this (`createProjectDashboardFragment`) since we're only making one fragment for the selected project!!

Reply:
u right